weather.js

```diff
@@ -5,17 +5,31 @@ dotenv.config();
 const apiURLPrefix = "http://api.openweathermap.org/";
 
 /**
- * Runs a conditional to distinguish if Zipcode or address string was entered.
+ * Get weather data based on whether Zipcode or address string was entered.
  * @param {string|number}input
  * @returns {Promise<void>}
  */
 async function getWeather(input) {
-    if (isNaN(input)) {
-        const data = await getCoordinates(input);
-        await getWeatherData(`lat=${data[0]["lat"]}&lon=${data[0]["lon"]}`);
-        return;
+    try {
+        formatCheck(input);
+        if (isNaN(input)) {
+            const data = await getCoordinates(input);
+            return printWeather(await getWeatherData(`lat=${data[0]["lat"]}&lon=${data[0]["lon"]}`));
+        }
+        printWeather(await getWeatherData(`zip=${input}`));
+    } catch (error) {
+        console.log(customErrorHandler(error));
+    }
+}
+
+/**
+ * Checks user input format
+ * @param {string|number}input
+ */
+function formatCheck(input) {
```

Comment:
I like to name methods that throw an exception 'validateXYZ'.
The validation implies that an exception will be thrown. Also the doblock should contain that.

When I see a method named `formatCheck` I think that its going to DO some formatting.
Sometimes I see methods called `checkInput` in which case I think its going to return a boolean.

Its all about having a descriptive verb and then adapting based on your team's general usage.